src/de/jost_net/JVerein/gui/dialogs/EigenschaftenAuswahlDialog2.java

```diff
@@ -0,0 +1,159 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.gui.dialogs;
+
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+
+import org.eclipse.swt.widgets.Composite;
+
+import de.jost_net.JVerein.gui.control.FilterControl;
+import de.jost_net.JVerein.server.EigenschaftenNode2;
+import de.willuhn.datasource.GenericIterator;
+import de.willuhn.jameica.gui.Action;
+import de.willuhn.jameica.gui.dialogs.AbstractDialog;
+import de.willuhn.jameica.gui.input.SelectInput;
+import de.willuhn.jameica.gui.parts.ButtonArea;
+import de.willuhn.jameica.gui.parts.TreePart;
+import de.willuhn.jameica.gui.util.LabelGroup;
+import de.willuhn.logging.Logger;
+
+/**
+ * Dialog, zur Auswahl von Eigenschaften eines Mitglied.
+ */
+public class EigenschaftenAuswahlDialog2
+    extends AbstractDialog<EigenschaftenAuswahlParameter2>
+{
+
+  private FilterControl control;
+
+  private SelectInput eigenschaftenverknuepfung;
+
+  private String defaults = null;
+
+  private boolean ohnePflicht;
+
+  private boolean verknuepfung;
+
+  private EigenschaftenAuswahlParameter2 param;
+
+  /**
+   * Eigenschaften oder Eigenschaftengruppen ausw�hlen
+   * 
+   * @param defaults
```

Comment:
Wenn die Parameter in der Doku drin stehen wäre eine kurze Beschreibung nicht schlecht.

Reply:
Ich sehe gerade, da gibt es noch mehr Parameter. Das stammt nicht von mir.
Da ich die auch noch ändere werde ich sie im nächsten PR beschreiben.